src/core/server/integration_tests/saved_objects/migrations/group3/actions/actions_test_suite.ts

```diff
@@ -237,85 +237,6 @@ export const runActionTestSuite = ({
         })
       );
     });
-    it('resolves left when cluster.routing.allocation.enabled is incompatible', async () => {
-      expect.assertions(3);
-      await client.cluster.putSettings({
-        body: {
-          persistent: {
```

Comment:
NIT: What's the difference between `src/core/server/integration_tests/saved_objects/migrations/group3/actions/actions.test.ts` and `src/core/server/integration_tests/saved_objects/migrations/group3/actions/actions_test_suite.ts` again? (why were the tests adapted in one and deleted in the other, it's trad vs serverless is that it?)